File: android/app/src/main/java/io/wassa/wisexampleproject/model/response/AnonymizationResponse.kt
package io.wassa.wisexampleproject.model.response

import com.google.gson.annotations.SerializedName
import io.wassa.wisexampleproject.model.JobStatus
import io.wassa.wisexampleproject.model.VehicleType

class AnonymizationResponse: Response() {
    @SerializedName("output_media") val outputMedia: String? = null
    @SerializedName("output_json") val outputJson: String? = null
}

File: dotnet/WIS/models/Responses/AnonymizationResponse.cs
using System.Text.Json.Serialization;

namespace Wassa.InnovationServices
{
    public class AnonymizationResponse : Response
    {
        [JsonPropertyName("output_media")]
        public string outputMedia { get; set; }
        [JsonPropertyName("output_json")]
        public string outputJson { get; set; }
    }
}